pipelines.yml

```diff
@@ -86,6 +96,42 @@ pipelines:
           onComplete:
             - update_commit_status buildJfrogCliPluginGit --context "$step_name"
 
+  - name: upgrade_plugin_on_core_release
+    configuration:
+      runtime:
+        type: image
+        image:
+          custom:
+            name: releases-docker.jfrog.io/jfrog-ecosystem-integration-env
+            tag: 1.0.0
```

Comment:
Wouldn't it be better to use ```latest```?
This applies to ```plugin_validation``` as well.

Reply:
Isn't it too risky to use latest for production?

Reply:
It's the latest release. Why would this be risky?

Reply:
Not the latest release, but the latest commit.